### ISSUE TITLE
Update instance names to match "ClassType" convention?

### ISSUE THREAD

**Issue by JordanMartinez** [purescript/purescript-foldable-traversable#132]
See [this comment in #131](https://github.com/purescript/purescript-foldable-traversable/pull/131#pullrequestreview-583961267) for more details.

**Comment by mhmdanas** [comment]
With PureScript 0.14.2 being released with automatic instance naming, I suspect that this is not needed anymore. However, I think 5.x versions of this library should probably be compatible with all versions of 0.14.x including the ones before 0.14.2, so this change should be made when PureScript 0.15.0 is released.

**Comment by JordanMartinez** [comment]
I'm not sure we should drop the instance names from core libraries. There's been a few issues that can arise with optional instance names when modules are being rebuilt. It might be safer to leave them in here.

**Comment by mhmdanas** [comment]
Good point; perhaps we should wait until all bugs related to that have been wrinkled out first.

**Comment by hdgarrood** [comment]
I don’t think we have any known bugs like that right now, but for me the reason to avoid this for the time being is that it would mean dropping support for earlier compiler versions, which we usually treat as a breaking change.
